src/components/app-configure/chain-management/AppChainManagementForm.vue

```diff
@@ -275,8 +275,16 @@ async function fetchChainIdUsingRPCUrl(rpcURL: string) {
               </button>
               <button
                 class="bg-[#FFFFFF] text-black w-[100px] p-2 rounded-md transition-opacity duration-500"
-                :disabled="!enableSave"
-                :class="[!enableSave ? 'opacity-5' : 'opacity-100']"
+                :disabled="
+                  !enableSave ||
+                  chainIDs.find((chain) => chain === formData.chainId)
+                "
+                :class="[
+                  !enableSave ||
+                  chainIDs.find((chain) => chain === formData.chainId)
+                    ? 'opacity-5'
+                    : 'opacity-100',
+                ]"
```

Comment:
I guess we can use the disabled state to dictate the styles to be applied, something like this

```vue
<button :disabled="!enableSave || chainIDs.find((chain) => chain === formData.chainId)" class="opacity-100 disabled:opacity-5"
```

Check if this works, so we don't need to repeat the condition as we scale.

Reply:
Sure, will check and try out.